app/commands/git/sync_track_docs.rb

```diff
@@ -0,0 +1,34 @@
+module Git
+  class SyncTrackDocs < Sync
+    include Mandate
+
+    def initialize(track)
+      super(track, track.synced_to_git_sha)
+    end
+
+    def call
+      return unless filepath_in_diff?("docs/config.json")
+
+      config = git_repo.read_json_blob(git_repo.head_commit, "docs/config.json")
+
+      config[:docs].each do |doc_config|
+        doc = Document.where(track: track).create_or_find_by!(
```

Comment:
Do we want to rescue the code in the loop to prevent one failing doc from failing everything?

Reply:
Yeah. Good idea. Although I'd like CI to protect us. And this should probably create an issue in the track if it hits that rescue block.